tests/link/pytorch/test_basic.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pytest
 
+import pytensor.tensor as pt
```

Comment:
We prefer to use explicit imports as a policy. In your case import expit and softplus directly